tests/integration/actions/actions.test.ts

```diff
@@ -0,0 +1,37 @@
+import { describe, expect, it, vi } from "vitest";
+import { render, screen } from "@testing-library/svelte";
+import userEvent from "@testing-library/user-event";
+import ClickOutsideOrEscapeActionComponent from "./ClickOutsideOrEscapeActionComponent.svelte";
+
+describe("Click outside or press escape action", () => {
+    it("When a component is created with the 'clickOutsideOrEscape' action and the user clicks outside, then the 'ClickedOutsideOrEscape' event is fired.", async () => {
```

Comment:
```suggestion
    test("When a component is created with the 'clickOutsideOrEscape' action and the user clicks outside, then the 'ClickedOutsideOrEscape' event is fired.", async () => {
```

As I understand it, there are two keywords for tests `it` and `test`. The reasons is to make a sentence with the keyword and the following sentence. So maybe use `test` instead of `it`?

This, of course, applies to all vitest tests.

Reply:
I adapted it.